test/Sync.t.sol

```diff
@@ -263,4 +180,45 @@ contract SyncTest is Test, Deployers, GasSnapshot {
         // The user lost 10 currency0, and can never claim it back.
         assertEq(currency0.balanceOf(address(this)), userCurrency0BalanceBefore - 10);
     }
+
+    function test_settle_failsWithNativeERC20IfNotSyncedInOrder(uint256 value) public noIsolate {
+        value = bound(value, 1, uint256(int256(type(int128).max / 2)));
+        vm.deal(address(this), value);
+        vm.deal(address(manager), value);
+        NativeERC20 nativeERC20 = new NativeERC20();
+
+        manager.sync(Currency.wrap(address(nativeERC20)));
+
+        // uint256 balanceBefore = address(this).balance;
```

Comment:
delete comment or move down to be part of the OZ attack comment

Reply:
Are you happe with the way it is handled at the bottom of the test?

https://github.com/vimageDE/v4-core/blob/1241d0c37c73ba766151c8fea4fdd30e8c5f49cb/test/Sync.t.sol#L221

Reply:
yeah i basically just think the entire commented out test should be together, and not interspersed among the new test

Reply:
Done